server/db/misc_store.go

```diff
@@ -0,0 +1,80 @@
+package db
+
+import (
+	"database/sql"
+	"fmt"
+
+	"github.com/mattermost/mattermost/server/public/model"
+
+	sq "github.com/mattermost/squirrel"
+)
+
+func (s *Store) KVGet(pluginID, key string, fromWriter bool) ([]byte, error) {
```

Comment:
Can we have a comment explaining why this KVGet (like you have for GetPost)?

Reply:
I think this is going away as soon as everything else is implemented.